src/interaction/InteractionManager.js

```diff
@@ -864,286 +804,184 @@ export default class InteractionManager extends EventEmitter
+     * Is called when the pointer button is pressed down on the renderer element
      *
      * @private
-     * @param {MouseEvent} event - The DOM event of a mouse button being pressed down
+     * @param {PointerEvent} originalEvent - The DOM event of a pointer button being pressed down
      */
-    onMouseDown(event)
+    onPointerDown(originalEvent)
     {
-        this.mouse.originalEvent = event;
-        this.eventData.data = this.mouse;
-        this.eventData._reset();
+        const events = this.normalizeToPointerData(originalEvent);
+
+        /**
+         * No need to prevent default on natural pointer events, as there are no side effects
+         * Normalized events, however, may have the double mousedown/touchstart issue on the native android browser,
+         * so still need to be prevented.
+         */
 
-        // Update internal mouse reference
-        this.mapPositionToPoint(this.mouse.global, event.clientX, event.clientY);
+        // Guaranteed that there will be at least one event in events, and all events must have the same pointer type
 
-        if (this.autoPreventDefault)
+        if (this.autoPreventDefault && (events[0].pointerType === 'touch' || events[0].pointerType === 'mouse'))
         {
-            this.mouse.originalEvent.preventDefault();
+            originalEvent.preventDefault();
         }
 
-        this.processInteractive(this.mouse.global, this.renderer._lastObjectRendered, this.processMouseDown, true);
+        const eventLen = events.length;
 
-        const isRightButton = event.button === 2 || event.which === 3;
-
-        this.emit(isRightButton ? 'rightdown' : 'mousedown', this.eventData);
-    }
-
-    /**
-     * Processes the result of the mouse down check and dispatches the event if need be
-     *
-     * @private
-     * @param {PIXI.Container|PIXI.Sprite|PIXI.extras.TilingSprite} displayObject - The display object that was tested
-     * @param {boolean} hit - the result of the hit test on the display object
-     */
-    processMouseDown(displayObject, hit)
-    {
-        const e = this.mouse.originalEvent;
-
-        const isRightButton = e.button === 2 || e.which === 3;
-
-        if (hit)
+        for (let i = 0; i < eventLen; i++)
         {
-            displayObject[isRightButton ? '_isRightDown' : '_isLeftDown'] = true;
-            this.dispatchEvent(displayObject, isRightButton ? 'rightdown' : 'mousedown', this.eventData);
-        }
-    }
+            const event = events[i];
 
-    /**
-     * Is called when the mouse button is released on the renderer element
-     *
-     * @private
-     * @param {MouseEvent} event - The DOM event of a mouse button being released
-     */
-    onMouseUp(event)
-    {
-        this.mouse.originalEvent = event;
-        this.eventData.data = this.mouse;
-        this.eventData._reset();
+            const interactionData = this.getIteractionDataForPointerId(event.pointerId);
```

Comment:
Should this be `getInteractionDataForPointerId` (note the n in Interaction)? This typo appears to be in a few places.